### PR TITLE
KOTS release notes v1.67.0

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

**Review by jonquil2002 — DISMISSED** [review]
lgtm if you include my commit.

File: docs/release-notes/rn-kubernetes-installer.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ toc_max_heading_level: 2
 
 # Kubernetes Installer Release Notes
 
+## Release v2022.03.21-0
+
+Released on March 11, 2022
```
Comment (erquhart) [inline docs/release-notes/rn-kubernetes-installer.md:9]: uuummmm....

**Review by erquhart — APPROVED** [review]

File: docs/enterprise/installing-embedded-cluster.md [hunk 2]
```diff
@@ -10,12 +10,16 @@ Before you install, ensure that you meet the system requirements. For more infor
...
 
-To install the admin console on a cluster created by the Kubernetes installer, run the installation script provided by the application developer.
+To install the admin console on a cluster created by the Kubernetes installer, run the installation script provided by the application vendor.
 
 ```bash
 curl -sSL https://kurl.sh/supergoodtool | sudo bash
 ```
 
+:::note
+With KOTS v1.67.0 and later, you can install a specific version of the application. Use the `appVersionLabel` flag and the version label for a particular version of your vendor's application. Example: `curl https://kurl.sh/supergoodtool | sudo bash -s appVersionLabel=3.1.0`.
```
Comment (erquhart) [inline docs/enterprise/installing-embedded-cluster.md:20]: flag should be `app-version-label`

File: docs/enterprise/installing-existing-cluster-online.md [hunk 3]
```diff
@@ -10,6 +10,9 @@ To start, run the command that was provided by the application vendor:
 ```shell
 kubectl kots install application-name
 ```
+:::note
+With KOTS v1.67.0 and later, you can install a specific version of the application. Use the `appVersionLabel` flag and the version label for a particular version of your vendor's application. Example: `kubectl kots install application-name --app-version-label=3.1.0`.
```
Comment (erquhart) [inline docs/enterprise/installing-existing-cluster-online.md:14]: flag should be `app-version-label`

**Review by ajp-io — APPROVED** [review]

**Pull request merged**